org-jira.el

```diff
@@ -240,8 +240,8 @@ instance."
   :group 'org-jira
   :type 'boolean)
 
-(defcustom org-jira-keywords-to-jira-status-alist nil
-  "Custom alist of 'org-mode' keyword stored in car and jira status in the cdr."
+(defcustom org-jira-jira-status-to-org-keyword-alist nil
+  "Custom alist of jira status stored in car and 'org-mode' keyword stored in cdr."
```

Comment:
The quotes should use 

    `org-mode'

so that emacs creates a link to org mode.